### PR TITLE
Make auth cookie jwt

### PR DESCRIPTION
**Pull request by cviecco**

### CODE REVIEW

File: cmd/smallpoint/main.go [hunk 1]
```diff
@@ -23,23 +23,28 @@ import (
...
+	TLSKeyFilename              string `yaml:"tls_key_filename"`
+	StorageURL                  string `yaml:"storage_url"`
+	OpenIDCConfigFilename       string `yaml:"openidc_config_filename"`
+	TemplatesPath               string `yaml:"templates_path"`
+	SMTPserver                  string `yaml:"smtp_server"`
+	SmtpSenderAddress           string `yaml:"smtp_sender_address"`
+	ClientCAFilename            string `yaml:"client_ca_filename"`
+	LogDirectory                string `yaml:"log_directory"`
+	ClusterSharedSecretFilename string `yaml:"cluster_shared_secret_filename"`
+	SharedSecrets               []string
```
[the remark this answers is not in the document]
Comment (cviecco) [reply cmd/smallpoint/main.go]: will be used to keep the shared secrets across multiple instances. This way they can verify the tokens as client pass them to each other.

File: cmd/smallpoint/main.go [hunk 2]
```diff
@@ -270,22 +268,23 @@ func main() {
...
+
 	http.Handle(creategroupWebPagePath, http.HandlerFunc(state.creategroupWebpageHandler))
 	http.Handle(deletegroupWebPagePath, http.HandlerFunc(state.deletegroupWebpageHandler))
 	http.Handle(creategroupPath, http.HandlerFunc(state.createGrouphandler))
 	http.Handle(deletegroupPath, http.HandlerFunc(state.deleteGrouphandler))
 
 	http.Handle(requestaccessPath, http.HandlerFunc(state.requestAccessHandler))
 	http.Handle(indexPath, http.HandlerFunc(state.mygroupsHandler))
-	http.Handle(authPath, simpleOidcAuth.Handler(http.HandlerFunc(state.mygroupsHandler)))
+	//http.Handle(authPath, simpleOidcAuth.Handler(http.HandlerFunc(state.mygroupsHandler)))
```
[the remark this answers is not in the document]
Comment (SLR9511) [reply cmd/smallpoint/main.go]: one more deletion here.

File: cmd/smallpoint/main.go [hunk 3]
```diff
@@ -40,6 +41,7 @@ type baseConfig struct {
```
Comment (SLR9511) [inline cmd/smallpoint/main.go]: you deleted the sharedsecrets from baseconfig?

Comment (cviecco) [reply cmd/smallpoint/main.go]: yes they are not needed yet, should be part of the pull request where it is implemented

**Pull request merged**